Forge.Forms/src/Forge.Forms/DynamicExpressions/MultiBinding.cs

```diff
@@ -38,9 +39,9 @@ public override BindingBase ProvideBinding(IResourceContext context)
 
         public override bool Equals(Resource other)
         {
-            if (other is MultiPropertyBinding resource)
+            if (other is MultiBinding resource)
             {
-                return PropertyPaths == resource.PropertyPaths
+                return Resources == resource.Resources
```

Comment:
This is a reference equality check, effectively making the whole function a reference comparison.
We could probably remove these overrides (Equals and GetHashCode) because deduping multi bindings would be quite difficult.

Reply:
See this https://github.com/WPF-Forge/Forge.Forms/blob/master/Forge.Forms/src/Forge.Forms/DynamicExpressions/DataBinding.cs for a reference equality version. I can't recall why some are like this and some like that.

The equality checks are being done at some point to reuse bindings when the same thing is being used multiple times in a string or boolean expression.